### DISCUSSION TITLE
[ LBSM Poll ]

### DISCUSSION THREAD

**Discussion by HbmMods** [HbmMods/Hbm-s-Nuclear-Tech-GIT#963] [General]
Less Bullshit Mode might get a couple new additions next week so I wanna know what the most annoying or boring parts of NTM's progression are

(vote now / no winners guaranteed / YMMV / may contain nuts / this product contains ingredients known to the state of california to cause cancer)

**Comment by SoB1nary** [comment]
comically large amount of steel needed for oil, also levis are too expensive beryllium wise and running an rbmk without them is a death sentence

**Reply by HbmMods** [reply]
i replaced half the beryllium blocks with cast iron plates

**Comment by masertank** [comment]
probably obtaining steel

**Reply by Nos74** [reply]
ditto, waiting a couple million years for the blast furnace to process shit is annoying

**Reply by HbmMods** [reply]
![image](https://user-images.githubusercontent.com/16674258/226198705-366089b0-504d-4d72-97e3-dc93c3c5256e.png)

hmmm early game blast furnace extensions

**Reply by masertank** [reply]
yea lmao

**Reply by masertank** [reply]
honestly i think there should just be a massive pear as an early game blast furnace extension
like it's just a massive pear

**Reply by HbmMods** [reply]
![2023-03-20_18 47 41](https://user-images.githubusercontent.com/16674258/226424508-13c43986-76f9-4244-9d4e-504ad18d07a0.png)

duende

**Comment by Nos74** [comment]
maskman's attacks breaking blocks is pretty shite, especially if you accidentally spawn him near your base in the early game when you can't kill him quickly

**Comment by funnyman8** [comment]
Finding a shit-load of rare earth ore just for niobium and zirconium, can only process the ore for one as well since desh blend can't be separated, and zirconium can only be gotten from centrifuging.

**Reply by HbmMods** [reply]
isn't zirconium a depth ore too

**Comment by LePeeperSauvage** [comment]
Crafting fuses requiring an assembly machine is annoying. The ionized particles are still really annoying especially if you're trying to make the ferric shcrab coils for a good hadron, which itself also needs a lot of it. And there are not enough silly cats

**Comment by ichbindaheim** [comment]
Arsenic production is a bit slow, maybe more items can be converted into oily scraps, Tar maybe?

**Comment by sdddddf80** [comment]
Post process. There is no diversified gameplay like the petrochemical production line and RBMK, just repetitive synthesis layer after layer, especially the synthesis of the spark battery succession. The fusionary Watz Plant is also boring as it is just like a bigger and more expensive diesel generator.

**Reply by HbmMods** [reply]
all of lategame being garbage is already well known, i'm working on it

**Reply by sdddddf80** [reply]
maybe the particle accelerator can learn something from the Quantum Minecraft Dynamics mod

**Reply by HbmMods** [reply]
i've never played QMD nor do i want to make the particle accelerator more complicated than it already is.

**Reply by Antineutrino-4444** [reply]
That will be AWESOME!!! HBM is now a much larger mod and have a very refined (like rp1 :>) and realisitc system
The late game could have more "future" nuclear theme into it, like some more advanced stuff with the standard models and so on
It can also be based on the current lore (possibly?) that can make HBM great again (without building a wall XD).
Anyways, glad to hear that lategame is being worked on!

PS: will the trello progress stuff keep getting updates? or im just bad at finding stuffs?

> all of lategame being garbage is already well known, i'm working on it

**Reply by HbmMods** [reply]
the trello page hasn't been updated in years and there's no real point in using it since often long planned things are scrapped or features get implemented right away with short notice, which means having a list of planned features doesn't really mean anything.

**Reply by OlorinIV** [reply]
> maybe the particle accelerator can learn something from the Quantum Minecraft Dynamics mod

or maybe we could craft 'satellite coils' launched into space, making an orbital hadron that runs virtually.

**Reply by Hinanium-Ion** [reply]
the watz fusion could be better. fusion reactor based on singularities is a awesome idea. but now the watz fusion is just a bid diesel generator.

**Comment by 70000hp** [comment]
RBMK rod depletion time config, would increase the amount of people reaching lategame tenfold

**Comment by NahidaSa326** [comment]
Can watz fusion reactor be changed to use singularity (including miniature black hole) as reaction energy for power generation? Different fuels are introduced for fusion (singularity only affects the efficiency and state of fusion).Fuel is fused through the singularity. Hydrogen-iron needs extra coolant or cold gel to cool itself. Additional electricity is introduced to constrain the singularity.

**Reply by HbmMods** [reply]
The Fusionary Watz, at least the way it is now, is definitely going to be removed. I'm not entirely sure what to put in its place progression-wise, possibly some new type of fusion reactor (perhaps an ICF?), and along it the Watz Supercompressor (visually similar to the current FWatz but it's a machine used to make singularities), power generation through singularities is probably going to be pushed back in progression, although I'll think about the details once I've figured out the current Watz Powerplant and whatever comes directly after that.

**Reply by NahidaSa326** [reply]
As for the watz fusion reactor, it will be a normal fusion reactor, because the fusion reactor will not generate electricity but will produce leafy green, ionized particles and thermonuclear ash in the later stage, and an additional plasma heater is needed, so I hope that these two fusion reactors can be combined and generate electricity in multiple modes, such as magnetic fluid power generation or heat exchange test and direct energy output.

**Reply by Hinanium-Ion** [reply]
maybe fusionary watz could not only be a generator, but also a machine prduce some advance item(similar to the normal fusion reactor?)

**Reply by 70000hp** [reply]
That is exactly what bob said fam

**Reply by NahidaSa326** [reply]
Agree that watz is not an engine but a machine for producing new substances.

**Comment by OlorinIV** [comment]
My idea of the FWatz rework came from Powder Toy, that game made hydrogen able to fuse all way down to iron under increasing pressure, temp, and gravitation conditions. And FWatz happened to be a singularity-driven gravity-confined fusion reactor.
Maybe the FWatz could also be a star-simulation kind of reactor in the future, being able to push the fusion chain further as we use stronger singularities at its core, and we can extract the resulting matter after shutting it down.
The fun of said machine should come from trying to maintain the 'star' as stable and economical as possible. Higher temperature and gravitation means better reactivity and longer reaction chain, the closer to the core the higher they are. Different singularities provide different heat and gravity stats and spark singularity gives pulsing stats. Reactions create heat and new matter and make the 'star' swell in size, possibly leading to meltdown if gravity isn't strong enough. New matter which haven't reached their reaction conditions became 'dead' matter and dulls 'star'. Sudden increase in reactivity may lead to helium flash, supernova, etc.
To make this thing actually useful, it should be able to push the fusion chain down further than iron, even to schrabidium but with efficiency loss. It would break its fuel into their consisting elements, accepting heavier elements as heat and gravity rises, such as carbon from coal, and oxygen and silicon from stone.

**Reply by masertank** [reply]
incredible idea, i can see this being applied to the mod

**Reply by NahidaSa326** [reply]
Bob may reintroduce his thermal system.

**Comment by NahidaSa326** [comment]
Can the problem of unstable block loading of RBMK reactor be solved? In some cases, when the block loading is incomplete, the reactor runs, but the cooling system is not fully started, resulting in the failure of cooling water supply, which leads to the problem of melting and explosion

**Comment by NahidaSa326** [comment]
![1680769424_69404_Lhgt](https://user-images.githubusercontent.com/128782236/230401292-58d759d6-d66b-4db8-965c-37f4a2b7a626.png)
This linear accelerator can produce a large amount of antimatter (it seems to use a bug?)

**Reply by HbmMods** [reply]
Antimatter has a momentum requirement of 900, a single schrabidium coil already has a coil strength of 250, assuming those are four gold coils (25 each), two neodymium (50 each) and two schrab coils (250 each) then that's exactly 900. Where exactly do you think is there an issue?

**Reply by OlorinIV** [reply]
The other side of this hadron.
![2023-04-06_15 46 37](https://user-images.githubusercontent.com/104282311/230519472-4ce9dc4a-0ed0-4cbf-bebe-93b4ff83c332.png)
Works without enclosing the analysis chamber.

**Reply by HbmMods** [reply]
Well yeah, since there are no checks part the second segment, you can leave it open like that.
